### ISSUE TITLE
Get FTS token balance- getBalance query

### ISSUE THREAD

**Issue by noshmody** [hashgraph/hedera-services#447]
**Summary**
Extend crypto getBalance HAPI message to include the balance of ONE FTS token held the account. 
Ensure support of pure **_hbar_** transfers and backward compatibility

**_HAPI Changes_** - 
The existing transfer list will remain unchanged.
Add a FTS Transfer list to the existing cryptoTransfer HAPI of the form  
message FTSTransferList {
	string  tokenName = 1;                     // user defined token name
	TransferList FTStransfers = 2;        // transferList of tokens of type tokenName
}

**Response Codes**:
**_Records_**:
**_Stats_**:
**_Fees_**:

**Comment by lbaird** [comment]
I don’t think this one should have a token transfer list, because no tokens are transferred.

I think the query shoood remain unchanged, but the reply should have two new fields: the token balance (an int64 balance of tinytokens), and the token type ID. Both should be missing if there is no token type associated with that account.

________________________________
